Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,996 @@
+        outputs_key_field="config",
+        outputs={"integration_config": integration_config},
+    )
+
+
+def get_list_of_labels(client: Client):
+    """Command to fetch list of label names based on company"""
+    labels = client.get_labels()
+    labels_formatted = [
+        {"No": index + 1, "Key": dt} for index, dt in enumerate(labels)
+    ]
+
+    table_name = "Labels"
+    headers = ['No', 'Key']
+    if labels_formatted:
+        readable_output = tableToMarkdown(table_name, labels_formatted, headers=headers)
+    else:
+        readable_output = tableToMarkdown(table_name, "No data found", headers=headers)
+
+    return CommandResults(
+        outputs_prefix="DSPM.Labels",
+        outputs_key_field="Key",
+        outputs=labels_formatted,
+        readable_output=readable_output,
+    )
+
+
+def dspm_get_risk_findings_command(client, args):
+    page = 0
+    headers = ["ID", "Rule Name", "Severity", "Asset Name", "Asset ID", "Status", "Project ID",
+               "Cloud Provider", "Cloud Environment", "First Discovered", "Compliance Standards"]
+    while True:
+        findings = dspm_get_risk_findings(client, args, page)
+        if not findings:
+            if page == 0:
+                demisto.info("No risks were fetched")
+                readable_output = tableToMarkdown("Risk Findings", [], headers=headers, headerTransform=pascalToSpace)
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.RiskFindings",
+                        outputs_key_field="id",
+                        readable_output=readable_output,
+                    )
+                )
+            break  # No more findings to fetch
+
+        readable_output = tableToMarkdown("Risk Findings", findings, headers=headers, headerTransform=pascalToSpace)
+        return_results(
+            CommandResults(
+                outputs_prefix="DSPM.RiskFindings",
+                outputs_key_field="id",
+                outputs=findings,
+                readable_output=readable_output,
+            )
+        )
+        page += 1
+
+
+def dspm_get_list_of_assets_command(client, args):
+    page = 0
+    headers = ["ID", "Project ID", "Project Name", "Name", "Cloud Provider", "Cloud Environment", "Service Type",
+               "Lifecycle", "Open Risks Count", "Open Alerts Count", "Encrypted", "Open To World",
+               "Tags", "Asset Dig Tags"]
+    while True:
+        assets = get_list_of_assets(client, args, page)
+        if not assets:
+            if page == 0:
+                readable_output = tableToMarkdown("List of assets", [], headers=headers, headerTransform=pascalToSpace)
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.Assets",
+                        outputs_key_field="id",
+                        outputs=[],
+                        readable_output=readable_output,
```

Comment:
```suggestion
                return_results(
                    CommandResults(
                        readable_output=readable_output,
```
in this case we need only the readable_output

Reply:
alright